cls/SourceControl/Git/Utils.cls

```diff
@@ -163,6 +163,16 @@ ClassMethod WarnInstanceWideUncommitted() As %Boolean
     quit $get(@..#Storage@("settings","warnInstanceWideUncommitted"),1)
 }
 
+ClassMethod EnvironmentName() As %String
+{
+    quit $get(@..#Storage@("settings","environmentName"),"")
```

Comment:
IMO this shouldn't be in a separate global - let's just have this call $System.Version.SystemMode